packages/cloud2edge/templates/NOTES.txt

```diff
@@ -13,9 +13,9 @@ You can check the current status by issuing the following command:
 Once all services are up and running, the output should look similar to this:
 
 NAME                                       {{ "READY   STATUS    RESTARTS   AGE" | indent ( len .Release.Name ) }}
-{{ .Release.Name }}-adapter-amqp-vertx-65cfb4d675-g5wn4       1/1     Running   0          3m15s
-{{ .Release.Name }}-adapter-http-vertx-66bd6bb89c-mng5t       1/1     Running   0          3m15s
-{{ .Release.Name }}-adapter-mqtt-vertx-765fcd578b-5rl7n       1/1     Running   0          3m15s
+{{ .Release.Name }}-adapter-amqp-65cfb4d675-g5wn4       1/1     Running   0          3m15s
+{{ .Release.Name }}-adapter-http-66bd6bb89c-mng5t       1/1     Running   0          3m15s
+{{ .Release.Name }}-adapter-mqtt-765fcd578b-5rl7n       1/1     Running   0          3m15s
```

Comment:
IMHO this breaks the indentation of the *STATUS* column, i.e. you will need to add *space* characters for the removed `vertx-` suffix ...

Reply:
sorry about the status - it seems that when I tested I was blind :D 
also added the respective properties(kafka.auth.sasl.jaas.clientUsers and kafka.auth.sasl.jaas.clientPasswords) to the values.yaml - is this sufficient or do you propose/prefer a different approach ?